src/Microsoft.VisualStudio.LiveShare.Razor/Guest/RazorGuestInitializationService.cs

```diff
@@ -0,0 +1,91 @@
+﻿//// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
@srivatsn this is the bit that enables us to download view imports ahead of time. Pointing this out so when you guys add the API you know what piece of ours is using it.